normflows/utils/nn.py

```diff
@@ -125,6 +130,54 @@ def forward(self, inputs):
         return out[..., self.inv_perm]
 
 
+class PeriodicFeaturesCat(nn.Module):
+    """
+    Converts a specified part of the input to periodic features by
+    replacing those features f with [sin(scale * f), cos(scale * f)].
+
+    Note that this decreases the number of features and their order
+    is changed.
+    """
+
+    def __init__(self, ndim, ind, scale=1.0):
+        """
+        Constructor
+        :param ndim: Int, number of dimensions
+        :param ind: Iterable, indices of input elements to convert to
+        periodic features
+        :param scale: Scalar or iterable, used to scale inputs before
+        converting them to periodic features
+        """
```

Comment:
"""                                                                                                                                                                                                                                                                       
        Constructor                                                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                  
        Args:                                                                                                                                                                                                                                                                     
          ndim (int): number of dimensions                                                                                                                                                                                                                                        
          ind (Iterable): indices of input elements to convert to periodic features                                                                                                                                                                                               
          scale (Scalar or iterable): used to scale inputs before converting them to periodic features                                                                                                                                                                            
        """